### ISSUE TITLE
(Orc) Blood Fury: Does not scale correctly, and at times give less AP than 25% of base (Also does not *stack* with Slayers Crest, Earthstrike, similar)

### ISSUE THREAD

**Issue by imaglcian** [slowtorta/turtlewow-bug-tracker#3338]
**Please, provide a clear description what the bug is:**

So since Blood Fury does scale with Strength for Warrior, and Strength/Agility for rogue, I wanted to test some numbers initially.
We had Arkimonday test Blood Fury with varying amounts of Attack Power, see the sample size here (This was not checked with BetterCharacterStats, but the standard Character Panel:

https://cdn.discordapp.com/attachments/992781235741016133/1101101509703368714/image.png

Additionally, here is how we tested each sample: https://streamable.com/lth7sz

- Blood Fury does not seem to be scaling properly by 25% of base AP, although we did not note down the amount of Strength before checking (Battle Shout was used in some of these samples, although not all.)
- Attack Power trinkets do not stack with Blood Fury.

**Steps to reproduce the behavior with as much detail as possible:**

Pop Blood Fury at different base AP values, get as big a sample size as possible. I could not recreate a graph for it, since the increased values were so jumbled. The value should always be the same.
Additionally reproduce each sample with Earthstrike, Jom Gabbar and Slayers Crest and see if mentioned trinket works with blood fury.

0. (If testing trinket) - Pop said trinket.
1. Check how much base AP you have.
2. Pop Blood Fury.
3. Check how much AP you have.
4. Check the difference in each numbers, % is required.
5. The increase should always be 25% of base AP (Before Battle Shout/Trinkets, but with Strength for Warriors, and Strength/Agility for Rogue)
6. Reproduce with different gear / Different base AP value.

**Expected behavior. Describe how it should work:**

- Blood Fury should only increase by 25% base AP (Including added AP from Str/Agi, not + AP)
- Blood Fury should also stack with trinkets like Slayers Crest, Earthstrike, Jom Gabbar (& More).

**Comment by TatkaS1** [comment]
Base attack power is the attack power you have before any gear or buff, the AP from whatever agility or strength you have at your level when naked. Blood Fury should be around +40AP at lvl 60 for orc iirc

**Comment by TatkaS1** [comment]
Base attack power is the white attack power you have before getting any AP from gear or buff, the AP from whatever agility or strength you have. Blood Fury should be around +40AP at lvl 60 without str/agi gear iirc (AP from str/agi adds to base)
*It shouldn't scale with any trinket buffs either

**Comment by imaglcian** [comment]
> Base attack power is the white attack power you have before getting any AP from gear or buff, the AP from whatever agility or strength you have. 
I accidentally wrote that trinkets didn't scale with Blood Fury, but I meant stack yeah.
I got some more brains into this, but it still would seem as if the scaling is incorrect, but I will have to redo my samples (As we tested them differently).
However Sample No. 3 and No. 7 were both done with Base AP only, no battle shout/trinkets up, and the value did not increase by 25%
